src/common/plugins/vuetify.ts

```diff
@@ -51,11 +51,13 @@ export const vuetify = createVuetify({
         variables: {
           'theme-on-purple': '#000',
           'theme-on-green': '#000',
+          'theme-on-pink': '#000',
```

Comment:
why do you set this variable?

Reply:
By design it looks like in dark mode we should always use black over a pink background.